packages/x-charts/src/ChartsTooltip/ChartsAxisTooltipContent.tsx

```diff
@@ -9,7 +9,7 @@ import { ChartSeriesDefaultized, ChartSeriesType } from '../models/seriesType/co
 import { AxisDefaultized } from '../models/axis';
 import { ChartsTooltipClasses } from './chartsTooltipClasses';
 import { DefaultChartsAxisTooltipContent } from './DefaultChartsAxisTooltipContent';
-import { isCartesianSeriesType } from './utils';
+import { isCartesianSeriesType } from '../internals/isCaterisan';
```

Comment:
File name is wrong

Reply:
Which one would you recomand?

I preferred to keep a single file for both `isCartesianSeries` and `isCartesianSeriesType` so I picked the common  part that is not the exact name of a function

Reply:
Oh, I don't mind much, the issue is that `isCaterisan` is the wrong spelling. Should be `isCartesian`